tests/Nox.Lib.Tests/Factories/Types/NoxTypeDateTimeRangeFactoryTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using FluentAssertions;
+using Nox.Factories.Types;
+using Nox.Lib.Tests.FixtureConfig;
+using Nox.Solution;
+
+namespace Nox.Lib.Tests.Factories.Types;
+
+public record DateTimeRangeDto(DateTimeOffset Start, DateTimeOffset End);
+public class NoxTypeDateTimeRangeFactoryTests
+{
+    [Theory, AutoMoqData]
+    public void CreateNoxType_FromDto_IsValid(NoxSolution noxSolution, EntityDefinitionFixture fixture)
```

Comment:
Can we add a test for null a value and an invalid value?

Reply:
Yes, tests added.